src/main/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiController.java

```diff
@@ -79,17 +78,14 @@ public ControlledGcpResourceApiController(
             .bucketName(body.getGcsBucket().getName())
             .build();
 
-    // This value is serialized in Stairway, so we avoid making a generic list.
     ControlledResourceIamRoleList privateRoles =
-        (ControlledResourceIamRoleList)
-            Optional.ofNullable(body.getCommon().getPrivateResourceUser())
-                .map(
-                    user ->
-                        user.getPrivateResourceIamRoles().stream()
-                            .map(ControlledResourceIamRole::fromApiModel)
-                            .collect(toCollection(ArrayList::new)))
-                .orElse(null);
-
+        Optional.ofNullable(body.getCommon().getPrivateResourceUser())
+            .map(
+                user ->
+                    user.getPrivateResourceIamRoles().stream()
+                        .map(ControlledResourceIamRole::fromApiModel)
+                        .collect(toCollection(ControlledResourceIamRoleList::new)))
```

Comment:
Not having to do a List cast in the `flightMap.get` is nice, but I'm a little worried about having a consistent way to serialize Lists in Stairway. This doesn't feel like a great long term solution for List serialization, and having only one kind of migration to do would be nice. Or is this the new preferred style to use everywhere?
If we do want to keep the ControlledResourceIamRoleList, consider adding a unit test for it that `flightMap.put -> toJson -> fromJson -> flightMap.get`, since that's the whole purpose of the class. But that might be just my paranoia about serialization.

Reply:
Fair point - defining a new type every time we serialize a generic is pretty heavy. If this is something we want to look at more broadly in Stairway, I think it's better to be consistent here